eox_nelp/signals/tasks.py

```diff
@@ -12,31 +12,17 @@
 from django.db.models import Q
 from eox_core.edxapp_wrapper.courseware import get_courseware_courses
 from eox_core.edxapp_wrapper.enrollments import get_enrollment
-from eox_core.edxapp_wrapper.grades import get_course_grade_factory
 from opaque_keys.edx.keys import CourseKey
 
+from eox_nelp.api_clients.certificates import ExternalCertificatesApiClient
 from eox_nelp.api_clients.futurex import FuturexApiClient
 from eox_nelp.edxapp_wrapper.course_overviews import CourseOverview
+from eox_nelp.signals.utils import _user_has_passing_grade
 
 courses = get_courseware_courses()
-CourseGradeFactory = get_course_grade_factory()
 logger = logging.getLogger(__name__)
 
 
-def _user_has_passing_grade(user, course_id):
```

Comment:
Ok this also brings some refactor.
https://github.com/eduNEXT/eox-nelp/blob/2b62c92f911e8638b981a4ea432ec490f9a1749a/eox_nelp/signals/utils.py#L52-L63

Reply:
yes,  Imoved some common method to an different file in order to use them in multiple places, the name starting with  _  could be consider as wrong since that have a private connotation, but I didn't make bigger this pr by adding name changes